modules/core/test/test_lpsolver.cpp

```diff
@@ -151,4 +151,20 @@ TEST(Core_LPSolver, issue_12337)
     //need to update interface: EXPECT_ANY_THROW(Mat1b z_8u; cv::solveLP(A, B, z_8u));
 }
 
+TEST(Core_LPSolver, issue_12343)
+{
+    Mat A = (cv::Mat_<double>(4, 1) << 3., 3., 3., 4.);
+    Mat B = (cv::Mat_<double>(4, 5) << 0., 1., 4., 4., 3.,
+                                       3., 1., 2., 2., 3.,
+                                       4., 4., 0., 1., 4.,
+                                       4., 0., 4., 1., 4.);
+
+    Mat z;
+    int result = cv::solveLP(A,B,z);
```

Comment:
I believe, we still need some normalization or auto-tuning of `SOLVELP_EPS` value.

- `cv::solveLP(A,B,z);` => `OK`
- `cv::solveLP(A,B*2,z);` => `OK`
- `cv::solveLP(A,B*10,z);` => `OK`
- `cv::solveLP(A,B*1e3,z);` => `OK`
- `cv::solveLP(A,B*1e4,z);` => `FAIL`

`A` can be multiplied by any positive scalar too.

---

BTW, use `2**k` multiplier to minimize calculation errors in normalization (if normalization is going to be used).